app/models/P5/departure/Consignment.scala

```diff
@@ -98,6 +97,10 @@ case class Consignment(
     _.map(_.conveyanceReferenceNumberToString).mkString("; ")
   )
 
+  val consignmentItems: Seq[ConsignmentItem] = HouseConsignment.foldLeft(Seq.empty[ConsignmentItem]) {
+    (x, y) => x ++ y.ConsignmentItem.map(_.copy(Consignor = y.Consignor))
```

Comment:
Could we give x and y more meaningful names here? Even if it's just `acc` and `house`. Not the easiest logic to follow